fuzzers/qemu_coverage/src/fuzzer.rs

```diff
@@ -253,7 +253,7 @@ pub fn fuzz() {
             &mut fuzzer,
             &mut state,
             &mut mgr,
-            options.timeout,
+            core::time::Duration::from_secs(options.timeout),
```

Comment:
I think we should parse this in `FuzzerOptions`

Reply:
done